faq.md

```diff
@@ -4,7 +4,7 @@ title: FAQ
 nav: faq
 ---
 
-Ask technical questions [on StackOverflow using the 'bazel' tag](https://stackoverflow.com/questions/tagged/bazel).
+If you have questions or need help, see the [Get Support](support.html) page.
 
 What is Bazel?
```

Comment:
opt: you could use headers (e.g. `# What is Bazel?`) and then you wouldn't need to pad the next line with `---`s

Reply:
Replaced all headers with `## Title`.